.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for submitting a pull request to ESLint. Before continuing, please be sure you've read over our guidelines:
+http://eslint.org/docs/developer-guide/contributing/pull-requests
+
+Specifically, all pull requests containing code require an issue (documentation-only pull requests do not require an issue). If this pull requests contain code, and there isn't yet an issue explaining why you're submitting this pull request, please stop and open a new issue first.
+
+Please answer all questions below.
+-->
+
+**What issue does this pull request address?**
+
+
+**What changes did you make? (Give an overview)**
+
+
+**Is there anything you'd like to reviewers to focus on?**
```

Comment:
Specifically, **all pull requests containing code require an accepted issue** (documentation-only pull requests do not require an issue). If this pull requests contain code, and there isn't yet an issue explaining why you're submitting this pull request, please stop and open a new issue first. If the issue is not accepted yet, please wait.